ext-src/packages/conan/ConanDependencies.ts

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ConanPackage } from "./ConanPackage";
+import { PackageDependencies } from "../PackageDependencies";
+import { PackageDependenciesHelper } from "../PackageDependenciesHelper";
+import { ComponentEntry } from "../../models/ComponentEntry";
+import { ScanType } from "../../types/ScanType";
+import { PackageDependenciesOptions } from "../PackageDependenciesOptions";
+import { ConanUtils } from "./ConanUtils";
+
+/**
+* @class ConanDependencies
+*/
+export class ConanDependencies implements PackageDependencies {
+  private theseAreTheLockFilesIKnow: Array<string> = ["composer.lock", "Cargo.lock", "Gemfile.lock", "yarn.lock", "renv.lock"];
```

Comment:
https://www.youtube.com/watch?v=8nvzEqsZIGo

Reply:
For the record, Conan generates arbitrary `.lock` files, over a predictable name like you see in this list, hence the need to track these, and exclude the stuff that isn't Conan, but also has `.lock` in the name.